static/js/message_edit_history.js

```diff
@@ -78,3 +79,8 @@ export function show_history(message) {
     overlays.open_modal("#message-edit-history");
     fetch_and_render_message_history(message);
 }
+
+export function initialize() {
+    const rendered_message_history = render_message_history();
```

Comment:
This probably deserves a comment noting that we should migrate this modal to be rendered when opened rather than once at startup and then its content edited by JavaScript.

(Or feel free to do that in a second commit here).

Reply:
Done.